### ISSUE TITLE
Values of custom annotations appear in the reports as tag

### ISSUE THREAD

**Issue by adlenton** [serenity-bdd/serenity-core#566]
Hi all,

as we are moving in the direction of using more custom tags in our testing project, I just figured out that the values of those custom tags are put into the serenity reports as tags - but not even separated but under another header.
I'm not entirely sure about the intended behavior (either not displayed at all or under their own header), but the current behavior seems at least buggy.

Code:
``
@WithTagValuesOf({"feature:premium features", "story:premium offers", "level:smoke", "level:integration"})
@UserName("GP1686")
public class CRMBeginnersPackageCampaignTest extends SimpleTestTemplateWithLogin {
``
![serenity reports tag 1](https://cloud.githubusercontent.com/assets/20105978/19229027/bc87e594-8eca-11e6-90fe-f9936c2eb27a.png)

**Comment by wakaleo** [comment]
Yes the tags marked with the @WithTagValueOf annotation appear in the "Related Tags" section - what behaviour were you expecting?

**Comment by wakaleo** [comment]
(As an aside, it is usually better to represent stories and features using requirements-based directory structure rather than with explicit tags.)

**Comment by adlenton** [comment]
Sure about the tags, but I was wondering if referring to the annotation @UserName, sorry for being unclear! Unfortunately pasted the wrong screenshot :(

**Comment by adlenton** [comment]
Can't edit comments on mobile, meh. 
So whatever I put in as value for the annotation @UserName gets put as story tag.

**Comment by adlenton** [comment]
Proper screenshot where you can now also see that it also puts the class names as tag in there (see the code from above)
![serenity reports tag 2](https://cloud.githubusercontent.com/assets/20105978/19233885/a6be50b8-8ee8-11e6-9736-7a3681698727.png)

**Comment by wakaleo** [comment]
This is indeed a bug. Custom tags like @Username should not appear in the reports at all. Is this what you would expect?

**Comment by adlenton** [comment]
Exactly, yeah :)

**Comment by sneharora** [comment]
In my case, I do need custom annotations to appaer in reports , but they are being ignored. can we enable this functionality via flag or something
